backend/app.py

```diff
@@ -789,6 +794,97 @@ def getFilteredType():
         logging.exception(f"Error retrieving reports.")
         return jsonify({"error": "Internal Server Error"}), 500
 
+@app.route("/api/reports/summarization/templates", methods=["POST"])
+def addSummarizationReport():
```

Comment:
some confusions here. We can meet to discuss to make sure this can streamline the report creation process with this.

Reply:
added a companyTickers param based on what we talked about in the meeting